uefi-raw/CHANGELOG.md

```diff
@@ -18,6 +18,8 @@
 - Added `UsbIoProtocol`.
 - Added `Usb2HostControllerProtocol`.
 - Added  `DevicePathProtocol::length()` properly constructing the `u16` value
+- Added `AllocateType`.
```

Comment:
We did a release recently, so these changes should now move to a new `## Added` section under `[Unreleased]`.